harmonic/install_macos_src.md

```diff
@@ -90,42 +82,18 @@ General dependencies:
 brew install assimp boost bullet cmake cppzmq dartsim@6.10.0 doxygen eigen fcl ffmpeg flann freeimage freetype gdal gflags google-benchmark gts ipopt jsoncpp libccd libyaml libzzip libzip nlopt ode open-scene-graph ossp-uuid ogre1.9 ogre2.3 pkg-config protobuf qt@5 qwt-qt5 rapidjson ruby tbb tinyxml tinyxml2 urdfdom zeromq
 ```
 
-`dartsim@6.10.0` and `qt@5` are not sym-linked. To use those dependencies when building
-`gz-physics7` and `gz-gui8`, run the following after installation:
-
-For Macs with Intel processors, add them to `/usr/local`:
+`dartsim@6.10.0` and `qt@5` are "keg only" Homebrew formulae and their paths must be explicitly configured before building Gazebo:
 
 ```bash
 # dartsim@6.10.0
-export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/usr/local/opt/dartsim@6.10.0
-export DYLD_FALLBACK_LIBRARY_PATH=${DYLD_FALLBACK_LIBRARY_PATH}:/usr/local/opt/dartsim@6.10.0/lib:/usr/local/opt/octomap/local
-export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/opt/dartsim@6.10.0/lib/pkgconfig
-# qt@5
-export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/usr/local/opt/qt@5
-```
-
-Note if you are on an ARM based Apple Silicon Mac machine, you will need to add them to `/opt/homebrew` instead:
+export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH:+$CMAKE_PREFIX_PATH:}`brew --prefix dartsim@6.10.0`
+export DYLD_FALLBACK_LIBRARY_PATH=${DYLD_FALLBACK_LIBRARY_PATH:+$DYLD_FALLBACK_LIBRARY_PATH:}`brew --prefix dartsim@6.10.0`/lib:`brew --prefix octomap`/local
+export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+$PKG_CONFIG_PATH:}`brew --prefix dartsim@6.10.0`/lib/pkgconfig
 
-```bash
-# dartsim@6.10.0
-export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/homebrew/opt/dartsim@6.10.0
-export DYLD_FALLBACK_LIBRARY_PATH=${DYLD_FALLBACK_LIBRARY_PATH}:/opt/homoebrew/opt/dartsim@6.10.0/lib:/opt/homebrew/opt/octomap/local
-export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/opt/homebrew/opt/dartsim@6.10.0/lib/pkgconfig
 # qt@5
-export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/homebrew/opt/qt@5
+export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH:+$CMAKE_PREFIX_PATH:}`brew --prefix qt@5`
```

Comment:
I'm not sure why you changed the syntax to `${CMAKE_PREFIX_PATH:+$CMAKE_PREFIX_PATH:}`. I think it's easier to read the original syntax. I also think it's a little easier to read the `$(...)` instead of backticks

```suggestion
export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:$(brew --prefix qt@5)
```

Reply:
`${CMAKE_PREFIX_PATH:+$CMAKE_PREFIX_PATH:}` is the unfortunate shell (Bash, Zsh) syntax for "append a : delimiter after the variable if it has a value". Otherwise if `CMAKE_PREFIX_PATH` is not set and you write `${CMAKE_PREFIX_PATH}:/opt/homebrew` you get `:/opt/homebrew`.